### ISSUE TITLE
does not support multiple projects?

### ISSUE THREAD

**Issue by lilydjwg** [neoclide/coc-rls#26]
It seems that it only spawns a rls for the first project Vim opens.

**Comment by chemzqm** [comment]
Yes, because rls doesn't support workspaceFolders, rls-vscode start multiple rls instances for different workspaceFolders which can't work with coc.nvim.
